x/pairing/keeper/unresponsive_provider_test.go

```diff
@@ -8,19 +8,14 @@ import (
 	"github.com/lavanet/lava/utils/lavaslices"
```

Comment:
add unit tests or modify existing ones to test all your added functionality:
1. cannot modify a stake entry delegation if a provider is frozen due to jail (checks delegate.go line 233)
2. check unfreeze TX not working for jailed providers (from the Msg server)
3. check jail mechanism: 1HR stakeAppliedBlock for first two jails, 24HR stakeAppliedBlock for 3rd jail, jail reset only if provider wasn't jailed for 24H

Reply:
done

Reply:
look at the new added test